### DISCUSSION TITLE
How to configure webhooks on Stripe's dashboard

### DISCUSSION THREAD

**Discussion by mpvosseller** [pay-rails/pay#678] [Q&A] [answered]
Are we supposed to manually set up / configure the webhook on Stripe's dashboard? And if so should we listen for all Stripe events or just some subset of the events?

The [webhook documentation](https://github.com/pay-rails/pay/blob/v3.0.24/docs/7_webhooks.md#routes) (see below) makes it sound like maybe there is a built in tool to configure the webhook but I get a 404 when trying to go to https://mydomain.com/pay/webhooks/stripe 

```
To configure webhooks on your payment processor, use the following URLs (with your domain):

Stripe - https://example.org/pay/webhooks/stripe
Braintree - https://example.org/pay/webhooks/braintree
Paddle - https://example.org/pay/webhooks/paddle
```

Thank you!

**Comment by excid3** [comment, marked as answer]
Yep, you need to add the webhook URLs to your payment processor account(s).
